.circleci/config.yml

```diff
@@ -51,9 +51,6 @@ jobs:
   deploy:
     docker:
       - image: circleci/python:2.7-stretch
-    environment:
-      - PYPI_USERNAME: nzlabplus-bot  # here be the name of the Government Innovation Service Lab Pypi account to automatically publish the OpenFisca-Aotearoa package to Pypi
```

Comment:
I guess these values are now set entirely through the CircleCI web interface. I recommend to document them in this file as well, as it can be a mystery for contributors that are not familiar with that service.

Reply:
Now documented in the shell script